spec/factories/course_assessment_categories.rb

```diff
@@ -2,7 +2,7 @@
   sequence(:course_assessment_category_weight) { |n| n }
   sequence(:course_assessment_category_title) { |n| "Category #{n}" }
   factory :course_assessment_category, class: Course::Assessment::Category do
-    course
+    course { create(:course) }
```

Comment:
I have to create a course here, otherwise `course.root_folder` will fail.
Another alternation I can think of would be specify a root_folder here.
@lowjoel  What do you think ?